--- a/debian/cosmic-greeter.pam	2025-10-02 13:27:46.999051962 +0200
+++ b/debian/cosmic-greeter.pam	2025-10-02 13:30:15.519431441 +0200
@@ -1,9 +1,9 @@
 #%PAM-1.0
 auth    requisite       pam_nologin.so
 auth	required	pam_succeed_if.so user != root quiet_success
-@include common-auth
+auth 	include 	system-login
 auth    optional        pam_gnome_keyring.so
-@include common-account
+account 	include 	system-login
 # SELinux needs to be the first session rule. This ensures that any 
 # lingering context has been cleared. Without this it is possible 
 # that a module could execute code in the wrong domain.
@@ -20,6 +20,6 @@
 session required        pam_limits.so
 session required        pam_env.so readenv=1
 session required        pam_env.so readenv=1 user_readenv=1 envfile=/etc/default/locale
-@include common-session
+session	include		system-login
 session optional        pam_gnome_keyring.so auto_start
-@include common-password
+session	optional	pam_elogind.so
+password	include 	system-login
+session optional pam_exec.so type=open_session /usr/bin/userctl set-environment cosmic-comp
+session required /lib/security/pam_limits.so

